Networking/WebSockets/WebSocketImpl.cc

```diff
@@ -246,8 +246,10 @@ namespace litecore { namespace websocket {
         // Body:
         if (_curMessageLength + length > _curMessage.size)
             return false; // overflow!
-        memcpy((void*)&_curMessage[_curMessageLength], data, length);
-        _curMessageLength += length;
+        if (length > 0) {
```

Comment:
The reason for this might become unclear later, so I would like to make it a habit of leaving a one line comment with a short reason, and the bug number 

e.g.
> CBL-2169: Attempting to read from a zero length slice throws an exception